### PR TITLE
Improve portfolio positions UI

### PR DESCRIPTION
**Pull request by vyorkin**

### CODE REVIEW

File: src/ui/pages/Overview/Positions/Positions.tsx [hunk 1]
```diff
@@ -407,35 +407,39 @@ function ProtocolHeading({
...
-          paddingBlock: 4,
-          paddingInline: 6,
-          backgroundColor: 'var(--neutral-200)',
-          borderRadius: 8,
-        }}
-      >
-        {`${formatPercent(relativeValue, 'en')}%`}
-      </UIText>
-    </HStack>
+    <VStack gap={8}>
```
Comment (everdimension) [inline src/ui/pages/Overview/Positions/Positions.tsx:410]: You wrapped one element in VStack which does nothing

File: src/ui/pages/Overview/Positions/Positions.tsx [hunk 2]
```diff
@@ -614,6 +618,17 @@ function PositionList({
               // overscan={5} // the library detects window edge incorrectly, increasing overscan just visually hides the problem
               items={items}
             />
+            {dappIndex !== preparedPositions.dappIds.length - 1 ? (
+              <div
+                style={{
+                  height: 1,
```
Comment (everdimension) [inline src/ui/pages/Overview/Positions/Positions.tsx:624]: 2

File: src/ui/pages/Overview/Positions/Positions.tsx [hunk 3]
```diff
@@ -614,6 +618,17 @@ function PositionList({
...
+                  width: '100%',
```
Comment (everdimension) [inline src/ui/pages/Overview/Positions/Positions.tsx:625]: unnecessary, and line isn't fullbleed in design

File: src/ui/pages/Overview/Positions/Positions.tsx [hunk 4]
```diff
@@ -614,6 +618,17 @@ function PositionList({
...
+                  backgroundColor: 'var(--neutral-200)',
+                  marginTop: 16,
+                  marginBottom: 8,
```
Comment (everdimension) [inline src/ui/pages/Overview/Positions/Positions.tsx:628]:
```suggestion
                  marginTop: 14,
                  marginBottom: 0,
```

Comment (vyorkin) [reply src/ui/pages/Overview/Positions/Positions.tsx]: ✅

File: src/ui/pages/Overview/Positions/Positions.tsx (hunk 3 above)
Comment (vyorkin) [reply src/ui/pages/Overview/Positions/Positions.tsx]: ✅

File: src/ui/pages/Overview/Positions/Positions.tsx (hunk 2 above)
Comment (vyorkin) [reply src/ui/pages/Overview/Positions/Positions.tsx]: ✅

File: src/ui/pages/Overview/Positions/Positions.tsx (hunk 1 above)
Comment (vyorkin) [reply src/ui/pages/Overview/Positions/Positions.tsx]: ✅

**Pull request merged**